invisible_cities/io/rwf_io.py

```diff
@@ -1,23 +1,57 @@
+import numpy  as np
 import tables as tb
 
+from typing  import Callable
+
 from .. reco import tbl_functions as tbl
 
 
-def rwf_writer(file,
+def rwf_writer(h5out           : tb.file.File          ,
                *,
-               group_name      : 'options: RD, BLR',
-               table_name      : 'options: pmtrwf, pmtcwf, sipmrwf',
-               compression     = 'ZLIB4',
-               n_sensors       : 'number of pmts or sipms',
-               waveform_length : 'length of pmt or sipm waveform_length'):
-    try:                       rwf_group = getattr          (file.root, group_name)
-    except tb.NoSuchNodeError: rwf_group = file.create_group(file.root, group_name)
+               group_name      :          str          ,
+               table_name      :          str          ,
+               compression     :          str = 'ZLIB4',
+               n_sensors       :          int          ,
+               waveform_length :          int          ) -> Callable:
```

Comment:
Curiously, that's not something that I changed but now that you mention it it shouldn't work. Must just be that we've always set everything explicitly or something. I'll change it to default to the NEW numbers

Reply:
You are right, it was already like that. I've only noticed it because you changed something else :laughing: 
If you want, leave it as is and we open a new PR fixing it everywhere. I don't think it should take any particular number.

Reply:
OK, I'll leave it for now